implementations/rust/ockam/ockam_transport_tcp/src/transport/lifecycle.rs

```diff
@@ -20,13 +20,32 @@ impl TcpTransport {
     /// # Ok(()) }
     /// ```
     #[instrument(name = "create tcp transport", skip_all)]
-    pub fn create(ctx: &Context) -> Result<Self> {
+    pub fn create(ctx: &Context) -> Result<Arc<TcpTransport>> {
```

Comment:
Same as in the udp, shouldn't this be named `get_or_create` ? Then the `create_new` could be just `create`